### PR TITLE
Check that decorator is not a string

### PR DESCRIPTION
**Pull request by davidbrochart**

### CODE REVIEW

**Comment by davidbrochart** [comment]
This would prevents issues like https://github.com/QuantStack/memestra/issues/14#issuecomment-633956813

**Comment by davidbrochart** [comment]
This fixes #14.
All tests are passing locally except one, which is the same as the one failing in https://github.com/davidbrochart/memestra/runs/706803043, revealed by GitHub Actions:
```
=================================== FAILURES ===================================
__________________________ TestCaching.test_defaults ___________________________

self = <test_caching.TestCaching testMethod=test_defaults>

    def test_defaults(self):
        tmpdir = tempfile.mkdtemp()
        try:
            os.environ['XDG_CONFIG_HOME'] = tmpdir
            cache = memestra.caching.Cache()
            key = memestra.caching.CacheKey(__file__)
            self.assertNotIn(key, cache)
            cache[key] = {}
            data = cache[key]
            self.assertEqual(data['version'], memestra.caching.Format.version)
>           self.assertEqual(data['name'], 'caching')
E           AssertionError: 'test_caching' != 'caching'
E           - test_caching
E           ? -----
E           + caching

tests/test_caching.py:41: AssertionError
```

**Comment by davidbrochart** [comment]
My mistake, it was a leftover of local files when I implemented GitHub Actions CI, all tests pass locally with this PR.

**Comment by davidbrochart** [comment]
@JohanMabille, @serge-sans-paille, ready to merge?

File: tests/notebook.py [hunk 1]
```diff
@@ -2,9 +2,9 @@
 from memestra import nbmemestra
 
 class NotebookTest(TestCase):
-    
+
     def test_nb_demo(self):
-        output = nbmemestra.nbmemestra("tests/misc/memestra_nb_demo.ipynb", ('decoratortest, deprecated'))
+        output = nbmemestra.nbmemestra("tests/misc/memestra_nb_demo.ipynb", ('decoratortest', 'deprecated'))
```
Comment (serge-sans-paille) [inline tests/notebook.py:7]: Good catch!

**Pull request merged**